media/css/m24/base.scss

```diff
@@ -86,8 +86,9 @@ a:link {
 
 .m24-c-cta {
     @include bidi(((background-position, left var(--cta-bg-position, 100%) top 100%, right var(--cta-bg-position, 100%) top 100%),));
+    box-decoration-break: clone;
     color: $m24-color-black;
-    display: inline-block;
```

Comment:
this display change does mean the underline gets a bit closer to text

height of box with inline-block display: 19.8
height of box with inline display: 18.5

more testing with an interior span is probably worthwhile, initial tests with this method did not fix the issue but I probably didn't move enough styles down to the span